### PR TITLE
Added a view schedule in manage employees

### PR DESCRIPTION
**Pull request by jaslatendresse**
I was initially going to make a whole new page for that but noticed the manage employees already listed all employee details. it only made sense to add a schedule view in the same style as what was already done.

### CODE REVIEW

File: views/manage_employees.php [hunk 1]
```diff
@@ -3,26 +3,25 @@
...
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
     <style>
-        td, th { padding:5px 15px 0 15px; }
+        td, th { padding:5px 13px 0 15px; }
     </style>
 </head>
-<body>
-<div class="col-md-12 well" style="position:absolute; top:10%; left:2.5%; width:95%;">
+<html>
+<div class="col-md-12 well" style="position:absolute; top:10%; left:5%; width:90%;">
```
Comment (samantha-kerr) [inline views/manage_employees.php]: Change this back to 2.5 and 95, like the rest of the app

**Comment by samantha-kerr** [comment]
Ive been taking care of all the ui

**Comment by jaslatendresse** [comment]
@samantha-kerr Oh sorry i thought it was from a previous pr from viktoriya. I don't know why those things have changed because I only added the select button and the function. I haven't touched the rest of the file at all.

**Comment by samantha-kerr** [comment]
Have you pulled master?
It should  be correct there

**Comment by jaslatendresse** [comment]
I think I'll just make a new pr

**Comment by samantha-kerr** [comment]
Why? Just pull master into your branch now, the changes will show

**Comment by samantha-kerr** [comment]
@jaslatendresse please remove the changes to the files that you have not done for this task

**Pull request merged**